code/game/objects/items/fireaxe.dm

```diff
@@ -88,3 +88,22 @@
 /obj/item/fireaxe/metal_h2_axe/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=5, force_wielded=23, icon_wielded="[base_icon_state]1")
+
+/obj/item/fireaxe/metal_h2_axe/update_icon_state()
+	. = ..()
+	icon_state = "metalh2_axe0"
+
+/obj/item/fireaxe/elder_atmosian_fireaxe
+	icon_state = "elder_axe0"
+	base_icon_state = "elder_axe"
+	name = "eldest fireaxe"
+	desc = "Upgrade of the metallic hydrogen axe; created by some madman with mysterious powers"
+
+/obj/item/fireaxe/elder_atmosian_fireaxe/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_unwielded=7, force_wielded=25, icon_wielded="[base_icon_state]1")
+	AddComponent(/datum/component/lifesteal, 15)
```

Comment:
Since this is an upgrade for the metallic hidrogen axe made with a compound that can be hard to make, i wanted to give it an interesting property that is lifesteal, the number is kinda eyeballed, but i can lower it to a more sane one, something like 5/6?

Reply:
Do you mind justifying the change? I get the new atmos content, but why would I make this/how much time is needed to make this/why is it good for the game as a whole?

Reply:
i see this as an endgame goal since it does require all your knowledge to make it